src/components/AccordionMenu/index.tsx

```diff
@@ -56,6 +57,7 @@ export class AccordionItemBase extends React.Component<ItemProps> {
   render() {
     const { children, expanded, title } = this.props;
     const contentId = makeItemContentId(title);
+    const expandPanel = expanded || title === infoPanelTitle;
```

Comment:
this feels a bit like magic functionality - that if the title of a panel happens to be the same string as the information panel it's expanded.  Can't we explicitly pass down an arg from where the `AccordionItem` is used in `renderFileInfo` to specify that this panel should be expanded instead?

Reply:
Good idea!

Reply:
[`6e875ac` (#1779)](https://github.com/mozilla/addons-code-manager/pull/1779/commits/6e875ac8df443c73a0054b6c00f028bec327ff52)